templates/CRM/Contribute/Form/Search/Common.tpl

```diff
@@ -160,11 +160,6 @@
 {include file="CRM/Campaign/Form/addCampaignToComponent.tpl" campaignContext="componentSearch"
 campaignTrClass='' campaignTdClass=''}
 
-<tr>
-  <td colspan="2">
-{include file="CRM/Contribute/Form/Search/ContributionRecur.tpl"}
```

Comment:
The new tpl file was missed in the commit. I have committed the file now. 

Can you please try again.